components/mixins/experience-event/experienceevent-bill-pay.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-bill-pay",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Bill Pay",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Bill pay steps such as start, save, failure, and complete.",
+  "definitions": {
+    "bill-pay": {
+      "properties": {
+        "xdm:bill-pay": {
+          "xdm:billPay": {
```

Comment:
Should we rename this to steps ? Given its current form, it will appear as (billPay.billPay)

Reply:
I can make this update.